### PR TITLE
Swordof bellenesse

### PR DESCRIPTION
**Pull request by nwg5817**
Just need some help with the Escaped Thief. He needs body parts & weapons.

Dungeon done.
Turn ins working.
Timers working.
Quest givers working.

### CODE REVIEW

**Comment by CrimsonMage** [comment]
, (32769, 107,       7616) /* ItemCurMana */
     , (32769, 108,       8000) /* ItemMaxMana */
and also
     , (32769, 136,       1) /* CriticalMultiplier */
     , (32769, 147,       1) /* CriticalFrequency */;
crit freq should never be 1
thats 100%
crit multi can be 1x but probably 2 or 2.5 in this case
maybe even 3
i'd check all your mana values to make sure they match max or even half (for some quests) or if focusing stone (empty)

**Comment by nwg5817** [comment]
> ```
>  , (32769, 107,       7616) /* ItemCurMana */
>  , (32769, 108,       8000) /* ItemMaxMana */
>  , (32769, 136,       1) /* CriticalMultiplier */
>  , (32769, 147,       1) /* CriticalFrequency */;
> ```
> 
> crit freq should never be 1
> thats 100%
> crit multi can be 1x but probably 2 or 2.5 in this case
> maybe even 3
> i'd check all your mana values to make sure they match max or even half (for some quests) or if focusing stone (empty)

Just pulled this from LSD and pcap. Didn't notice. What should it be?

**Comment by CrimsonMage** [comment]
Crit freq is usually between 0.1 (10%) and 0.35 (35%) -- This varies based on other stats of the item to not make an item extremely over powered vs other pieces.
Instances like this, usually 0.2 crit freq and 2.5 crit multi would work, aim low, its easier to buff something than it is to nerf it (people don't usually report OP items, they report under achieving items though.)

**Comment by nwg5817** [comment]
> Crit freq is usually between 0.1 (10%) and 0.35 (35%) -- This varies based on other stats of the item to not make an item extremely over powered vs other pieces.
> Instances like this, usually 0.2 crit freq and 2.5 crit multi would work, aim low, its easier to buff something than it is to nerf it (people don't usually report OP items, they report under achieving items though.)

What if I remove those measures? Does biting strike increase that number?

**Comment by CrimsonMage** [comment]
Biting strike is that value. Removing those will remove the Crushing Blow/Biting strike off the item.
Also you dropped the mana to 800, i'd check the mana on all the items to ensure the current int matches max int unless otherwise specified in the data. Pcaps are to help fill in gaps of the data but only show what was sent over the wire, some things sent over the wire were basic values to simply display ID data in the panel.

File: Database/Patches/2006-03-DownTwistingPaths/9 WeenieDefaults/32769 Replica Sword of Bellenesse.sql [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+     , (32769,  45,          3) /* DamageType - Slash, Pierce */
+     , (32769,  46,          2) /* DefaultCombatStyle - OneHanded */
+     , (32769,  47,          6) /* AttackType - Thrust, Slash */
+     , (32769,  48,         45) /* WeaponSkill - LightWeapons */
+     , (32769,  49,         50) /* WeaponTime */
+     , (32769,  51,          1) /* CombatUse - Melee */
+     , (32769,  53,        101) /* PlacementPosition - Resting */
+     , (32769,  93,       1044) /* PhysicsState - Ethereal, IgnoreCollisions, Gravity */
+     , (32769, 106,        300) /* ItemSpellcraft */
+     , (32769, 107,       800) /* ItemCurMana */
```
Comment (CrimsonMage) [inline Database/Patches/2006-03-DownTwistingPaths/9 WeenieDefaults/32769 Replica Sword of Bellenesse.sql]: This should be at 8000 as well as max - you accidentally removed a `0`
http://ac.yotesfan.com/weenies/items/32769

Comment (nwg5817) [reply Database/Patches/2006-03-DownTwistingPaths/9 WeenieDefaults/32769 Replica Sword of Bellenesse.sql]: https://asheron.fandom.com/wiki/Replica_Sword_of_Bellenesse

I don't know why but it seems the pcap was off.

Comment (CrimsonMage) [reply Database/Patches/2006-03-DownTwistingPaths/9 WeenieDefaults/32769 Replica Sword of Bellenesse.sql]: Fixed the wiki to match the pcap - pcap is correct.

File: Database/Patches/2006-03-DownTwistingPaths/9 WeenieDefaults/32833 Escaped Thief.sql [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+INSERT INTO `weenie_properties_create_list` (`object_Id`, `destination_Type`, `weenie_Class_Id`, `stack_Size`, `palette`, `shade`, `try_To_Bond`)
+VALUES (32833, 2,  28612,  0, 93, 0, False) /* Create Amuli Coat (6046) for Wield */
+     , (32833, 2,  414,  0, 93, 0, False) /* Create Amuli Leggings (6047) for Wield */
+     , (32833, 2,  25648,  0, 93, 0, False) /* Create Amuli Leggings (6047) for Wield */ 
+     , (32833, 2,  415,  0, 93, 0.9818, False) /* Create Amuli Leggings (6047) for Wield */ 	
+     , (32833, 2,  25646,  0, 93, 0, False) /* Create Amuli Leggings (6047) for Wield */ 	 
+     , (32833, 2,  81,  0, 93, 0, False) /* Create Amuli Leggings (6047) for Wield */ 	 	
+     , (32833, 2,  115,  0, 93, 0, False) /* Create Amuli Leggings (6047) for Wield */
+     , (32833, 9, 32832,  1, 0, 1, False) /* Create  (32832) for ContainTreasure */;
```
Comment (ziang4891) [inline Database/Patches/2006-03-DownTwistingPaths/9 WeenieDefaults/32833 Escaped Thief.sql]: Missing newline

**Comment by ziang4891** [comment]
The various body part texture properties should probably be left of to allow randomization of the NPC/mob looks that aren't proscribed by some other means

**Pull request closed without merging**